lib/active_record_event_publisher/configuration.rb

```diff
@@ -0,0 +1,25 @@
+module ActiveRecordEventPublisher
+  class Configuration
+    attr_accessor :aws_region, :aws_secret_access_key, :aws_access_key_id,
+      :log, :enabled, :queue_url
+
+    alias_method :enabled?, :enabled
+    alias_method :log?, :log
+
+    def log
+      @log || false
+    end
+
+    def aws_region
+      @aws_region || raise(ArgumentError, "#{self.class} aws_region required")
+    end
+
+    def aws_access_key_id
+      @aws_access_key_id || raise(ArgumentError, "#{self.class} aws_secret_key required")
+    end
+
+    def aws_secret_access_key
+      @aws_secret_access_key || raise(ArgumentError, "#{self.class} aws_access_key required")
+    end
```

Comment:
I think we should update the messaging on L18 and L22 to be consistent with the naming of expected keys.

From `aws_secret_key` to `aws_access_key_id`
From `aws_access_key` to `aws_secrety_access_key`